lib/fog/aws/requests/compute/stop_instances.rb

```diff
@@ -16,9 +16,13 @@ class Real
         #     * TODO: fill in the blanks
         #
         # {Amazon API Reference}[http://docs.amazonwebservices.com/AWSEC2/latest/APIReference/ApiReference-query-StopInstances.html]
-        def stop_instances(instance_id, force = false)
+        def stop_instances(instance_id, options = {})
           params = Fog::AWS.indexed_param('InstanceId', instance_id)
-          params.merge!('Force' => 'true') if force
+          params.merge!('Force' => 'true') if options['force']
```

Comment:
For backwards compatibility, could you tweak this to check to see if options is a boolean value, and if so print a warning and then automatically convert it to the appropriate hash? I think that should strike a good balance of adding the new functionality while also preventing a breaking change for existing users. Does that make sense?

Reply:
It does, I shall take a look at it

Reply:
Ok, added the change

Reply:
Looks great, thanks!